### PR TITLE
Fix onnxrt calibration issue

### PR DESCRIPTION
**Pull request by mengniwang95**
## Type of Change

bug fix

## Description

Fix onnxrt calibration issue

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Comment by mengniwang95** [comment]
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/5546/

File: neural_compressor/adaptor/ox_utils/calibration.py [hunk 1]
```diff
@@ -440,8 +440,9 @@ def get_weight_tensors_calib_range(self):
                 ),
             )
             calibrator = CALIBRATOR["minmax"]()  # use minmax method to calibrate initializer tensors
-            calibrator.collect(initializer_tensor)
-            weight_tensors_calib_range[initializer_tensor_name] = [list(calibrator.calib_range)]
+            if initializer_tensor.flatten().size > 0:
```
Comment (hshen14) [inline neural_compressor/adaptor/ox_utils/calibration.py:443]: do you know why the tensor size would be 0?

Comment (mengniwang95) [reply neural_compressor/adaptor/ox_utils/calibration.py:443]: initializer of some ops could be empty, for example Resize op:
image

**Comment by mengniwang95** [comment]
@chensuyue UT failed due to other fwks

**Pull request merged**